data-collection/src/main/java/refactoringml/db/Yes.java

```diff
@@ -1,34 +1,40 @@
 package refactoringml.db;
 
+import org.hibernate.annotations.Type;
 import refactoringml.util.RefactoringUtils;
-
 import javax.persistence.*;
-import java.util.Calendar;
 
+//TODO: create a Baseclass for both Yes and No, as they share a lot of logic
 @Entity
-@Table(name = "yes", indexes = {@Index(columnList = "project_id"), @Index(columnList = "refactoringType"), @Index(columnList = "refactoring"), @Index(columnList = "isTest")})
+@Table(name = "yes", indexes = {@Index(columnList = "project_id"), @Index(columnList = "refactoringLevel"), @Index(columnList = "refactoring"), @Index(columnList = "isTest")})
 public class Yes {
-
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private long id;
 
-	@ManyToOne
+	@ManyToOne(targetEntity = CommitMetaData.class)
```

Comment:
You don't need the targetEntity here. (which seems to be wrong, as it's pointing to the wrong class)